services/ai-navigator-app/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+displayName: DKP AI Navigator
+description: DKP AI Navigator is D2iQ’s cool chatbot tool - TODO CHANGE
+category:
+  - tools
+type: platform
```

Comment:
should we make this `type: internal`? Or do we want it to show up in the UI?

Reply:
It should show up in the UI, yes, but it should always be deployed into the `kommander` namespace.
We will build a hack to let it only show up on the management cluster workspace in the first place and later build a proper solution as we might need this for other apps as well. This is a different namespace but is much more accurate than having it available in every workspace.

Does your suggestion regarding the `$(releaseNamespace}` is still valid in this case?

Reply:
yep the `$(releaseNamespace}` var will still work. Hardcoding the namespace will work as well but i'd recommend using the variable since there wouldn't be any cons to doing so

Reply:
Thanks it works :)